README.md

```diff
@@ -72,6 +72,7 @@ The object literal follows the format:
 	cookie: 				// session cookie configuration
 	getUserString:			// method to return a string id for a user
 
+	noAccessLogs: false,			// disables access logs
```

Comment:
@jwcampbell42 You could make it an enable flag that defaults to true when not configured.